Tribler/community/market/community.py

```diff
@@ -164,6 +165,7 @@ def __init__(self, *args, **kwargs):
         self.matchmakers = set()
         self.request_cache = RequestCache()
         self.cancelled_orders = set()  # Keep track of cancelled orders so we don't add them again to the orderbook.
+        self.config = TriblerConfig()
```

Comment:
Please do not create a new instance of `TriblerConfig`. Instead, use the configuration inside the `tribler_session` object (`tribler_session.config`). **make sure to check whether `tribler_session` exists or not!**

Reply:
Fixed.